ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Cleaner.java

```diff
@@ -98,16 +98,24 @@ public void run() {
           handle = txnHandler.getMutexAPI().acquireLock(TxnStore.MUTEX_KEY.Cleaner.name());
           startedAt = System.currentTimeMillis();
           long minOpenTxnId = txnHandler.findMinOpenTxnIdForCleaner();
+          long minTxnIdSeenOpen = txnHandler.findMinTxnIdSeenOpen();
```

Comment:
could we skip this extra db call if we have `metastore.txn.use.minhistorylevel=false`